keps/sig-multicluster/3335-statefulset-slice/kep.yaml

```diff
@@ -0,0 +1,39 @@
+title: StatefulSet Slice
+kep-number: 3335
+authors:
+  - "@pwschuurman"
+owning-sig: sig-multicluster
+participating-sigs:
+  - sig-apps
+  - sig-storage
+status: provisional
```

Comment:
This needs to be merged as `implementable` to go into the enhancement freeze

Reply:
This still holds.